tests/api/routes/experiment.test.js

```diff
@@ -134,30 +134,10 @@ describe('tests for experiment route', () => {
       },
     ];
 
-    const result = {
-      cellSizeDistribution: {
-        M: {
-          filterSettings: {
-            M: {
-              minCellSize: {
-                N: '10800',
-              },
-              binStep: {
-                N: '200',
-              },
-            },
-          },
-          enabled: {
-            BOOL: false,
-          },
-        },
-      },
-    };
-
     request(app)
       .put('/v1/experiments/someId/processingConfig')
       .send(newData)
-      .expect(200, result)
+      .expect(200)
```

Comment:
You don't check for the validity of the resulting data?

Reply:
No, because we are effectively testing a mock here. We mock out this function, then we test that it returns what the mock should return. What we should be testing instead is if the data complies with the spec, which is what this test suite is for.